standard/structs.md

```diff
@@ -502,6 +504,50 @@ Automatically implemented properties ([§15.7.4](classes.md#1574-automatically-i
 
 > *Note*: This access restriction means that constructors in structs containing automatically implemented properties often need an explicit constructor initializer where they would not otherwise need one, to satisfy the requirement of all fields being definitely assigned before any function member is invoked or the constructor returns. *end note*
 
+A *property_declaration* ([§15.7.1](classes.md#1571-general)) for an instance property in a *struct_declaration* may contain the *property_modifier* `readonly`. However, a static property shall not contain that modifier.
+
+It is a compile-time error to attempt to modify the state of an instance struct variable via a readonly property declared in that struct.
+
+It is a compile-time error for an automatically implemented property having a `readonly` modifier, to also have a `set` accessor.
+
+It is a compile-time error for an automatically implemented property in a `readonly` struct to have a `set` accessor.
+
+An automatically implemented property declared inside a `readonly` struct need not have a `readonly` modifier, as its `get` accessor is implicitly assumed to be readonly.
+
+It is a compile-time error for a property to have a `readonly` modifier on both the `get` and the `set` accessors.
+
+It is a compile-time error to have a `readonly` modifier on a property itself as well as on either of its `get` and `set` accessors.
+
+If the `get` accessor has a `readonly` modifier, the `set` shall exist and shall not have that modifier.
+
+### §cands-diffs-methods Methods
+
+A *method_declaration* ([§15.6.1](classes.md#1561-general)) for an instance method in a *struct_declaration* may contain the *method_modifier* `readonly`. However, a static method shall not contain that modifier.
+
+It is a compile-time error to attempt to modify the state of an instance struct variable via a readonly method declared in that struct.
+
+Although a readonly method may call a sibling, non-readonly method, or property or indexer get accessor, doing so results in the creation of an implicit copy of `this` as a defensive measure.
```

Comment:
We haven't defined "sibling" anywhere in the spec. I suggest replacing it with "may call a ... **declared in the same `struct` declaration**" (emphasis mine for suggested text.